config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
@durandom - Thank you for adding me to the discussion. If the change is visible in the UI, which it looks like it will be, then we can add a step and a note to the procedure on adding RHV providers to call out this behaviour.

Would that work?